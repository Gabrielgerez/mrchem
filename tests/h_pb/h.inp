{
"world_prec": 1.0e-4,
"world_size": 5,
"MPI": {
  "numerically_exact": true
},
"Molecule": {
  "charge": -1,
  "coords": "H 0.0 0.0 0.0"
},
"WaveFunction": {
  "method": "pbe0",
  "environment": "pcm_pb"
},
"PCM": {
  "SCRF": {
    "kain": 6,
    "max_iter": 100,
    "dynamic_thrs": false
  },
  "Cavity": {
    "spheres": "0 2.645616384 1.0 0.0 0.2"
  },
  "Solvent":{
    "Permittivity": {
      "epsilon_in": 1.0,
      "epsilon_out": { "static": 78.4},
      "formulation": "exponential"
    },
    "DebyeHuckelScreening": {
      "ion_strength": 0.25,
      "ion_radius": 0.0,
      "ion_width": 0.2
    }
  }
},
"SCF": {
  "run": false,
  "guess_type": "sad_gto"
}
}
