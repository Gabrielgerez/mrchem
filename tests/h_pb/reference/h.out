                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         unknown                                  ***
***         Git commit hash    unknown                                  ***
***         Git commit author  unknown                                  ***
***         Git commit date    unknown                                  ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : unknown
 Git commit hash       : unknown
 Git commit author     : unknown
 Git commit date       : unknown

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (1 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 6
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                              -1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                             Solvation Cavity
---------------------------------------------------------------------------
 Formulation             :                                     exponential
 Dielectric constant     :            (in)                        1.000000
                         :           (out)                       78.400000
---------------------------------------------------------------------------
    N      R_0  Alpha Beta Sigma      Radius         x         y         z
---------------------------------------------------------------------------
    0   2.6456  1.00  0.00  0.20  ->  2.6456  0.000000  0.000000  0.000000
===========================================================================


===========================================================================
              Square of the Debye-Huckel screening parameter
---------------------------------------------------------------------------
 Formulation             :                                     exponential
 Screening function value:            (in)                        0.000000
                         :           (out)                        0.087032
---------------------------------------------------------------------------
    N      R_0 Sigma      Radius         x         y         z
---------------------------------------------------------------------------
    0   2.6456  0.20  ->  2.6456  0.000000  0.000000  0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999999944384e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.925860375949
 E-N energy              :            (au)                 -1.914553894864
 Coulomb energy          :            (au)                  1.220280348194
 Exchange energy         :            (au)                 -0.152520844705
 X-C energy              :            (au)                 -0.488533147613
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Reaction energy (el)    :            (au)                 -0.375288469231
 Reaction energy (nuc)   :            (au)                  0.190909814206
 Reaction energy (tot)   :            (au)                 -0.184378655025
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -0.784755632271
 Nuclear energy          :            (au)                  0.190909814206
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.938458180651e-01
                         :      (kcal/mol)             -3.726438769686e+02
                         :        (kJ/mol)             -1.559141981237e+03
                         :            (eV)             -1.615936792593e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.116572346446
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.233144692892
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                              -1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.925860375949
 E-N energy              :            (au)                 -1.914553894864
 Coulomb energy          :            (au)                  1.220280348194
 Exchange energy         :            (au)                 -0.152520844705
 X-C energy              :            (au)                 -0.488533147613
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Reaction energy (el)    :            (au)                 -0.375288469231
 Reaction energy (nuc)   :            (au)                  0.190909814206
 Reaction energy (tot)   :            (au)                 -0.184378655025
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -0.784755632271
 Nuclear energy          :            (au)                  0.190909814206
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.938458180651e-01
                         :      (kcal/mol)             -3.726438769686e+02
                         :        (kJ/mol)             -1.559141981237e+03
                         :            (eV)             -1.615936792593e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.116572346446
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -0.233144692892
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Nuclear vector          :       -0.000000       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :       -0.000000       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 51s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
